py_zipkin/zipkin.py

```diff
@@ -241,6 +241,10 @@ def __init__(
                 "use the timestamp and duration parameters."
             )
 
+        if self.zipkin_attrs_override and self.sample_rate is not None:
+            raise ZipkinError(
+                'sample_rate should not be set when zipkin_attrs are provided')
```

Comment:
Also add a section in DEPRECATIONS.rst on how to do the resampling now that this is gone.

Reply:
done